scratch-vm/src/extensions/noise/index.js

```
@@ -0,0 +1,200 @@
+    /** @param {number} seed */
+    function setSeed(seed) {
+        noise.SetSeed(seed);
+    }
+
+    /** @param {number} frequency */
+    function setFrequency(frequency) {
+        noise.SetFrequency(frequency);
+    }
+
+    /** @param {number} octaves */
+    function setFractalOctaves(octaves) {
```

ESLint

[error] Missing space before function parentheses.

(space-before-function-paren)